src/process-effects.js

```diff
@@ -18,6 +18,15 @@ module.exports = tagFunction => {
 	// create the component, which will save new effects to the effect queue
 	const tagResult = tagFunction()
 
+	if (Array.isArray(tagResult)) {
+		throw new TypeError('Tram-One: Sorry, Tram-One does not currently support array returns. Wrap components in an element before returning.')
+	}
```

Comment:
This could be eventually resolved by #173 , for now, an error should notify people that support doesn't exist (yet).